File: One Task/Task 1/c programming/21.c
// Armstrong Number Check

#include <stdio.h>
#include <math.h>

int main() {
    int n = 1634, i = 0, sum = 0, rem = 0;
    int temp = n, n2 = n;
    while(n > 0) {
        n /= 10;
        i++;
    }
    while(n2 > 0) {
        rem = n2 % 10;
        sum += pow(rem, i);
        n2 /= 10;
    }
    printf("%d", sum);

    if(temp == sum) {
        printf("%d is Armstrong Number", temp);
    }else {
        printf("%d is Not a Armstrong Number", temp);
    }
    return 0;
}

File: PHP/core/week-1/c-programs/programs/21.php
<?php
    $num = 153;
    $sum = 0;
    $temp = $num;

    while ($temp != 0) {
        $remainder = $temp % 10;
        $sum += pow($remainder, 3);
        $temp = (int)($temp / 10);
    }

    if ($sum == $num) {
        echo "$num is an Armstrong number.";
    } else {
        echo "$num is not an Armstrong number.";
    }
?>